omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientIdentifierResource1_8.java

```diff
@@ -110,7 +110,6 @@ public void setIdentifierType(PatientIdentifier instance, PatientIdentifierType
 			PatientIdentifierType pit = Context.getPatientService().getPatientIdentifierTypeByName(name);
 			if (pit != null) {
 				instance.setIdentifierType(pit);
-				reset = true;
```

Comment:
is this flag not used in any place in the entire class?

Reply:
Could you kindly point where?

Reply:
@sacull,Kindly review

Reply:
> is this flag not used in any place in the entire class?

This is a local variable, so its scope is only a method, not an entire class. In addition, it is changed at the very end of this method, so the method ends after the change and access to this variable is lost. So why waste CPU cycles to change this variable?